### PR TITLE
fix: [AXIMST-656] Course unit - Fixed space between xblocks

### PR DESCRIPTION
**Pull request by PKulkoRaccoonGang**
[YouTrack](https://youtrack.raccoongang.com/issue/AXIMST-656/unitxblock-Space-between-xblocks-must-be-reduced)

### CODE REVIEW

File: src/course-unit/CourseUnit.jsx [hunk 1]
```diff
@@ -129,6 +129,7 @@ const CourseUnit = ({ courseId }) => {
             <Layout.Element>
               {currentlyVisibleToStudents && (
                 <AlertMessage
+                  className="course-unit__alert"
```
Comment (monteri) [inline src/course-unit/CourseUnit.jsx:132]: 👍

File: src/course-unit/CourseUnit.jsx [hunk 2]
```diff
@@ -140,7 +141,7 @@ const CourseUnit = ({ courseId }) => {
                   courseId={courseId}
                 />
               )}
-              <Stack gap={4} className="mb-4 course-unit__xblocks">
+              <Stack className="mb-4 course-unit__xblocks">
```
Comment (monteri) [inline src/course-unit/CourseUnit.jsx:144]: 👍 
good class namings

File: src/course-unit/course-xblock/CourseXBlock.jsx [hunk 3]
```diff
@@ -76,12 +76,17 @@ const CourseXBlock = ({
 
   return (
     <div ref={courseXBlockElementRef} {...props}>
-      <Card as={ConditionalSortableElement} id={id} draggable className="mb-1">
+      <Card
+        as={ConditionalSortableElement}
+        id={id}
+        draggable
+        componentStyle={{ marginBottom: 0 }}
```
Comment (monteri) [inline src/course-unit/course-xblock/CourseXBlock.jsx:83]: Will `className=""mb-0` work. It is preferable to avoid inline styles

Comment (PKulkoRaccoonGang) [reply src/course-unit/course-xblock/CourseXBlock.jsx:83]: Unfortunately no

**Pull request merged**